layouts/partials/rbac-permissions-table.html

```diff
@@ -49,14 +49,16 @@
     <tbody>
       <tr>
         <td>
-          <a href="/logs/guide/logs-rbac-permissions/#logs_live_tail"><code>logs_live_tail</code></a>
+          <a href="/logs/guide/logs-rbac-permissions/#logs_live_tail">Logs Live Tail</a><br>
+          (<code>logs_live_tail</code>)
```

Comment:
worth noting: I never understood why this section is broken out separately and refers to these as "legacy" and "superseded" permissions. Out of scope for this particular task I think but I'd like to understand if the Logs team can help us clarify and simplify these.

Reply:
yeah i don't really get it either